arch/arm/core/aarch32/mpu/CMakeLists.txt

```diff
@@ -5,3 +5,9 @@ zephyr_library()
 zephyr_library_sources(                             arm_core_mpu.c)
 zephyr_library_sources_ifdef(CONFIG_CPU_HAS_ARM_MPU arm_mpu.c)
 zephyr_library_sources_ifdef(CONFIG_CPU_HAS_NXP_MPU nxp_mpu.c)
+
+if (CONFIG_CPU_CORTEX_R)
+zephyr_library_include_directories(cortex_a_r)
+else ()
```

Comment:
This could be an else if, as well. Potentially adding an else with an error message, also.